src/LibraryManager.Vsix/Commands/RestoreCommand.cs

```diff
@@ -72,6 +72,8 @@ public static void Initialize(Package package, OleMenuCommandService commandServ
 
         private async Task ExecuteAsync(object sender, EventArgs e)
         {
+            Telemetry.TrackUserTask("RestoreCommand");
```

Comment:
Same to above comment. TrackOperation() ?  "Run-RestoreCommand" ?

Reply:
This is a menu command invoked by the use. Should it be tracked as an operation?